test/functional/wallet_balance.py

```diff
@@ -257,5 +259,35 @@ def test_balances(*, fee_node_1=0):
         assert_equal(self.nodes[0].getbalance(minconf=0), total_amount + 1)  # The reorg recovered our fee of 1 coin
 
 
+        def test_lastprocessedblock_json_field():
```

Comment:
nit: no need to make this a function, because it is only called once. Could inline?